Makefile

```
@@ -18,6 +18,12 @@ GO_ENVIRONMENTS ?=
 # Wait4X output name
 WAIT4X_BINARY_NAME ?= wait4x
 
+# Change output to .exe for windows
+GOOS ?= $(shell go env GOOS)
+ifeq ($(GOOS),windows)
+WAIT4X_BINARY_NAME := ${WAIT4X_BINARY_NAME}.exe
+endif
```

```suggestion
ifeq (windows,$(filter windows,$(shell go env GOOS) $(TARGETOS)))
WAIT4X_BINARY_NAME := ${WAIT4X_BINARY_NAME}.exe
endif
```

The problem has two sides, building from local machines and building from pipeline. With my suggestion, it'll work on the a machine, but it won't work on the pipeline. I have to add something (make the TARGETOS available) to Dockerfile. I'll do that in another pull request.